src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -233,6 +269,8 @@ public Locomotive click(WebElement element) {
         return this;
     }
 
+    public Locomotive click(AccessibilityIdentifier id) { return click(waitForElementByAccessibilityIdentifier(id)); }
```

Comment:
Add all these methods to the `Conductor` interface.

Reply:
Added